### PR TITLE
typescript definition file

### PR DESCRIPTION
**Pull request by rhyek**

### CODE REVIEW

**Comment by brendanashworth** [comment]
@rhyek thank you for taking the time to put this together. Does this definition belong [with DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped) instead? This would allow imports like `require('@types/generate-password')`.

**Comment by rhyek** [comment]
It depends on the project's owner, tbh. If you include the type definitions inside this repo, it means users get the definitions automatically instead of having to do the extra step of something like `npm install @types/generate-password". Up to you entirely.

**Comment by rhyek** [comment]
Bit of a formatting/escaping issue there. The point is to skip doing `npm install @types/generate-password`.

File: package.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "version": "1.3.0",
   "description": "Easy library for generating unique passwords.",
   "main": "main.js",
+  "types": "types/generate.d.ts",
```
Comment (brendanashworth) [inline package.json]: Could you move this file to `src/generate.d.ts` please?

**Comment by crutchcorn** [comment]
This would be very nice to have in a release. Is there any word on progress @rhyek?

**Comment by brendanashworth** [comment]
@crutchcorn thanks for the ping— I'll make the changes I requested and publish a new release over the weekend.

**Comment by brendanashworth** [comment]
Merged in https://github.com/brendanashworth/generate-password/commit/ef2ded6195ef72ee364172d1ff2c4d107ffe2821, will be putting out a new release soon.

**Comment by brendanashworth** [comment]
@crutchcorn this is now available in the latest release, v1.4.0.

**Pull request closed without merging**